### PR TITLE
Changes suggested by Petr Novak

### PR DESCRIPTION
**Pull request by HeikoTheissen**

### CODE REVIEW

**Review by uhlmannm — APPROVED** [review]

File: vocabularies/Hierarchy.xml [hunk 1]
```diff
@@ -117,7 +117,7 @@ the following collections of hierarchy nodes are distinguished:
         <Property Name="DistanceFromRoot" Type="Edm.Int64" Nullable="true">
           <Annotation Term="Core.Description" String="Number of ancestors an occurrence of a node has in the limited hierarchy" />
           <Annotation Term="Core.LongDescription">
-            <String>This equals the number of ancestors in the sub-hierarchy as well as in the unlimited hierarchy.</String>
+            <String>This equals the number of ancestors in the sub-hierarchy, if the request involves one.</String>
```
Comment (ThomasChadzelek) [inline vocabularies/Hierarchy.xml:120]: What does "one" refer to here?

File: vocabularies/Hierarchy.xml [hunk 2]
```diff
@@ -177,9 +177,13 @@ The template actions themselves cannot be invoked.</String>
         <Annotation Term="Common.IsInstanceAnnotation" />
         <Annotation Term="Core.Description" String="Instance annotation on the result of an `$apply` query option containing the number of matching nodes after hierarchical transformations" />
         <Annotation Term="Core.LongDescription">
-          <String>The service designates a subset of the `$apply` result as "matching nodes".
-          This subset is the output set of the `filter` or `search` transformation that occurs as the fourth parameter
-          of the last `ancestors` transformation or occurs nested into it.</String>
+          <String>The service MAY designate a subset of the `$apply` result as "matching nodes".
+          For requests following [this pattern](#RecursiveHierarchyType), this subset is the output set of the
```
Comment (ThomasChadzelek) [inline vocabularies/Hierarchy.xml:181]: Not sure how #RecursiveHierarchyType defines a pattern, but never mind.

**Review by ThomasChadzelek — APPROVED** [review]

**Review by uhlmannm — APPROVED** [review]

**Comment by HeikoTheissen** [comment]
Will be merged after the V4 Tree meeting today (2023-09-08T15:00:00+02:00).

**Review by ThomasChadzelek — APPROVED** [review]

File: vocabularies/Hierarchy.xml (hunk 2 above)
Comment (ThomasChadzelek) [reply vocabularies/Hierarchy.xml]: I did click the link :-)

**Pull request merged**